tinker/e_announcements/__init__.py

```diff
@@ -318,4 +340,21 @@ def get_title_and_message(form):
 
     # TODO e-announcements by role (someday)
 
+    @route("/search", methods=['POST'])
+    def search(self):
+        data = self.base.dictionary_encoder.encode(json.loads(request.data))
+        selection = data['selection']
+        title = data['title']
+        date = data['date']
+        try:
+            date = datetime.datetime.strptime(date, "%a %b %d %Y")
+
+        except:
+            # Set start and end to be false so that hasDates is set to false
```

Comment:
this comment doesn't make sense here? probably was copy-pasted?

Reply:
Removed the comment